### PR TITLE
Add Stateless token store

### PR DESCRIPTION
**Pull request by cheungpat**
connects #74

### CODE REVIEW

**Comment by cheungpat** [comment]
Do not merge yet, I am trying to add a test case for user info with the token valid since field

**Comment by cheungpat** [comment]
Updated the pull request with the added test case

File: authtoken/authtoken.go [hunk 1]
```diff
@@ -162,6 +178,8 @@ func InitTokenStore(config Configuration) Store {
 		store = NewFileStore(config.Path, config.Expiry)
 	case "redis":
 		store = NewRedisStore(config.Path, config.Prefix, config.Expiry)
+	case "stateless":
```
Comment (rickmak) [inline authtoken/authtoken.go]: `jwt`? Seem expose the implementation to developer will save use from explanation on how we do the stateless store.

File: authtoken/stateless.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		return Token{}, err
+	}
+
+	token := Token{}
+	r.setTokenFromClaims(claims, &token)
+	token.AccessToken = signedString
+	return token, nil
+}
+
+// Get tries to read the specified access token from redis store and
```
Comment (rickmak) [inline authtoken/stateless.go]: `redis` store?

**Comment by cheungpat** [comment]
@rickmak updated the PR

**Pull request merged**